docs/releases/5.2.txt

```diff
@@ -150,7 +150,8 @@ Email
 Error Reporting
 ~~~~~~~~~~~~~~~
 
-* ...
+* The attribute :attr:`.SafeExceptionReporterFilter.hidden_settings` now
+  treats values as sensitive if their key name include ``AUTH``.
```

Comment:
```suggestion
  treats values as sensitive if their key name includes ``AUTH``.
```

Reply:
Or keep everything singular:  "treats a value as sensitive if its key name includes ``AUTH``."

Also, we might improve on "key name", since it's not always the case that users interact with their settings as a dictionary.

Maybe:

"Settings including ``AUTH`` in their name are now treated as sensitive by the attribute ...`"

Reply:
@jacobtylerwalls I opted for using "if their name includes..." (removing `key`) because I specifically want to avoid mentioning `settings` since a common use case for this filter is to sanitize request.META which is not about settings at all (but header names).